course-definition.yml

```
@@ -251,7 +251,7 @@ stages:
 
       - Just like the previous stage, you can hardcode `+PONG\r\n` as the response for this stage. We'll get to parsing
        client input in later stages.
-      - The two PING commands will be sent using the same connection. We'll get to handling multiple connections in later stages.
+      - The multiple PING commands will be sent using the same connection. We'll get to handling multiple connections in later stages.
```

```suggestion
      - The PING commands will be sent using the same connection. We'll get to handling multiple connections in later stages.
```

(multiple is redundant here since we say "commands" anyway)